docs/src/main/sphinx/connector/jdbc-common-configurations.fragment

```diff
@@ -51,4 +51,11 @@ connector:
     JDBC query. Using a large timeout can potentially result in more detailed
     dynamic filters. However, it can also increase latency for some queries.
     Defaults to `20s`.
+* - `statistics.enabled`
+  - Populate table statistics and use the [cost-based
```

Comment:
Lets confirm what this actually does .. from what I understand it does NOT populate statistics as stored in the underlying data source. Instead it creates statistics data on the fly based on the data read in the query or so .. thats quite different from how this work with object storage connectors and also NOT using the statistics from the data source if they are available. Lets make sure we get this right

Reply:
We fetch table statistics from the underlying jdbc data source and use that to extrapolate costs for all the operations on top of the scan.

Reply:
Ok .. so it depends on the statistics being there in the jdbc data source and that might or might not be the case. And how users go about making sure those stats are there depends on the JDBC data source. So next question ... can Trino get those stats updated for some data sources  (with ANALYZE) or do we rely on the native tools?

Reply:
JDBC connectors don't support statistics collection in Trino, we have to run ANALYZE in the underlying DB